src/components/ZipSearchField.jsx

```diff
@@ -0,0 +1,50 @@
+export default function ZipSearchField(props) {
+
+    const zipcodeApi = async (zipcode) => {
+        // put try catch block here
+        // API_BASE_URL specifies the API Base URL
+        const API_BASE_URL = "https://ctp-zip-code-api.onrender.com";
+        //try block to test if api call gives valid response or error
+        try{
+            // using a concatonated string to create the APU URL and then use fetch to call the API
+            const response = await fetch(API_BASE_URL + "/zip/" + zipcode);
```

Comment:
```suggestion
            const response = await fetch(`${API_BASE_URL}/zip/${zipcode}`);
```

also works and may be easier to read.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals